OpenTabletDriver.Desktop/Migration/SettingsMigrator.cs

```diff
@@ -10,21 +12,20 @@ public static class SettingsMigrator
         public static Settings Migrate(Settings settings)
         {
             // Output mode
-            MigrateNamespace(settings.OutputMode);
-            
-            // Bindings
-            if (settings.TipButton is PluginSettingStore tipStore)
-                MigrateNamespace(tipStore);
+            settings.OutputMode = SafeMigrateNamespace(settings.OutputMode, new PluginSettingStore(typeof(AbsoluteMode)));
 
-            while(settings.PenButtons.Count < Settings.PenButtonCount)
-                settings.PenButtons.Add(null);
-            foreach (PluginSettingStore store in settings.PenButtons)
-                MigrateNamespace(store);
+            // Bindings
+            settings.TipButton = SafeMigrateNamespace(settings.TipButton, new PluginSettingStore(
+                new MouseBinding
+                {
+                    Property = nameof(Plugin.Platform.Pointer.MouseButton.Left)
+                }
+            ));
```

Comment:
If the `DriverDaemon.DefaultSettings` gets moved, this should be done instead.
```suggestion
            // Bindings
            settings.TipButton = SafeMigrateNamespace(settings.TipButton, Settings.Default.TipButton);
```